UI/ChatWnd.cpp

```diff
@@ -195,13 +195,13 @@ void MessageWndEdit::FindGameWords() {
             m_game_words.insert(UserString(entry.second->Name()));
     }
     // add ship hulls
-    for (const auto& entry : GetPredefinedShipDesignManager()) {
-        if (entry.second->Hull() != "")
-            m_game_words.insert(UserString(entry.second->Hull()));
+    for (const auto& design : GetPredefinedShipDesignManager().GetOrderedShipDesigns()) {
```

Comment:
I'm confused about this whole block of code... why is it only adding hulls / parts that are present in designs in the predefined manager... I'd change this to add all parts / hulls. Not really relevant to this PR, though...